src/foam/dao/JDBCConnectionSpec.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/* A JDBC Connection Specification to add to the system context */
```

Comment:
Add comment inside the model instead of as a comment so that it's available to our tools.
Ex.:
documentation: 'A JDBC Connection Specification to add to the system context.'

Reply:
OK